app/controllers/questions_controller.rb

```diff
@@ -70,11 +183,32 @@ def log_time
   end
 
   def next
-    setup = Setup.take
-    week = setup.weeks.find_by_number(params[:week_number])
-    RecommenderSystem::Recommender.setup(current_user.id,week.id)
-    best = RecommenderSystem::HybridRecommender.new.get_best
-    los = LearningObject.find(best[0])
-    redirect_to action: "show", id: los.url_name
```

Comment:
Mas pravdu, v tomto pripade to naozaj nema zmysel. A bude to tak vzdy, ze pre kazdy test bude najviac 10 otazok? Taketo obmedzenie vyplyva z implementacie alebo sa to bude dat aj menit v administracii?

Ono idealne by bolo dat moznost nastavit, kolko otazok v ramci testu ma byt a ci chceme odporucat alebo nie - ale to samozrejme komplikuje situaciu. Takze takto to prehanat urcite nemusime :)